src/zenml/integrations/kserve/__init__.py

```diff
@@ -31,7 +31,7 @@ class KServeIntegration(Integration):
 
     NAME = KSERVE
     REQUIREMENTS = [
-        "kserve==0.9.0",
+        "kserve>=0.9.0",
```

Comment:
I would make this still fixed in the upper bound in case they release 0.11.0 that breaks our integration

```suggestion
        "kserve>=0.9.0,<=10",
```